### DISCUSSION TITLE
On select event?

### DISCUSSION THREAD

**Discussion by protricker** [easepick/easepick#27] [Q&A] [answered]
Hello. How can i detect changes in my datefield input after i applied picker on it?
I guess i need something like onSelect on picker object or something like that. I use RangePlugin.
Thank you!

**Comment by kl3sk** [comment, marked as answer]
Maybe this [discution](https://github.com/easepick/easepick/discussions/7) may help

**Reply by protricker** [reply]
Thanks a lot!
